### PR TITLE
chore(setup): `lerna` for monorepo package versioning

### LINKED ISSUE #48: ⚙️ Setup: `lerna` to manage monorepo package versioning
It would be nice to have an efficient way to maintain the same version number for all packages.

- https://lerna.js.org/docs/introduction

### PR DESCRIPTION
**Pull request by xamdoo**
### *What* change does this PR introduce?

This PR sets up Lerna for managing multiple packages within a single repository
### *Why* was this change needed?
Lerna would handle shared dependencies among packages within the monorepo reducing duplication and simplifying installation

- Closes #48 

### Checklist

- [X] 📝 I have reviewed my code prior to submission
- [X] ❔ Issue number has been linked (if applicable)
- [ ] 🧪 Tests have been added or updated (if applicable)
- [ ] 📖 Documentation has been updated (if applicable)
- [ ] 📸 Screenshots added (if applicable)

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "version": "0.0.0"
```
Comment (RickyC0626) [inline lerna.json]: **chore:** Let's bump to version `0.0.1`

**Review by RickyC0626 — CHANGES_REQUESTED** [review]
Remember to update the working directories in GitHub Actions workflows too.

File: packages/backend/package.json
Comment (RickyC0626) [inline packages/backend/package.json:1]: **chore:** Set version to `0.0.1` here as well.

Comment (xamdoo) [reply packages/backend/package.json:1]: does this apply to both frontend and backend packages or just the backend?

Comment (RickyC0626) [reply packages/backend/package.json:1]: Oh right, both. I think an easy way to do this is use the `lerna version` command with the flags `--no-changelog` and `--no-git-tag-version`. We can run the same command without flags during actual release.

**Review by RickyC0626 — DISMISSED** [review]
Looks good!

**Review by jaysabe — CHANGES_REQUESTED** [review]
Looks like we'll need to address merging conflicts before proceeding with the merge to dev.

**Comment by RickyC0626** [comment]
Since we have new updates to the original files, we probably have to move the directories to `packages/` again. `git mv` will be helpful for those. Perhaps 1 commit for each directory, as separate PRs. After renaming we can apply the `lerna` changes on top.

**Review by RickyC0626 — APPROVED** [review]

**Review by CeeJayyy007 — APPROVED** [review]
**praise:** Awesome work!

**Pull request merged**